src/styles/_catalog.scss

```diff
@@ -1,15 +1,20 @@
+$card-size: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-size);
+  justify-content: center;
 
   max-width: 944px;
   margin: 0 auto;
```

Comment:
you dont need `max-width` as well as `margin: 0 auto`

Reply:
fixed